config/routes.rb

```diff
@@ -1,17 +1,20 @@
 Rails.application.routes.draw do
 
-
   devise_for :users
   get 'welcome/index'
```

Comment:
This can be removed as `welcome#index` is available at the root path (see line 9).